types/dat.gui/index.d.ts

```diff
@@ -91,3 +117,12 @@ export class GUIController {
     listen(): GUIController;
     remove(): GUIController;
 }
+
+declare var index: {
+    gui: {
+        GUI: typeof GUI
+    },
+    GUI: typeof GUI
+};
+
+export default index;
```

Comment:
> Proof that implementation uses only export default: https://github.com/dataarts/dat.gui/blob/master/src/dat/index.js#L31

[Their build system, however](https://github.com/dataarts/dat.gui/blob/master/build/dat.gui.js#L2519), seems to have other plans.